graphql-apollo-server-tutorial.md

```diff
@@ -289,7 +285,7 @@ const resolvers = {
 };
 {{< /highlight >}}
 
-You may have noticed the first argument called `parent` as well. For now, you don't need to worry about it. Later, it will be showcased where it could be used potentially in your resolvers. Now, to make the example more realistic, you can extract a map of sample users and return a user based on the `id` which is used as a key in the extracted map:
+The first argument called `parent` as well, but you shouldn't worry about it for now. Later, it will be showcased where it can be used in your resolvers. Now, to make the example more realistic, extract a map of sample users and return a user based on the `id` used as a key in the extracted map:
```

Comment:
"The first argument called `parent` as well," -> "The first argument is called `parent`,"